easybuild/easyconfigs/s/Subread/Subread-2.0.0-GCC-10.2.0.eb

```diff
@@ -0,0 +1,44 @@
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+
+easyblock = 'MakeCp'
+
+name = 'Subread'
+version = '2.0.0'
+
+# HTTPS is not valid
+homepage = 'http://%(namelower)s.sourceforge.net/'
+description = """High performance read alignment, quantification and mutation discovery"""
+
+toolchain = {'name': 'GCC', 'version': '10.2.0'}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = ['%(namelower)s-%(version)s-source.tar.gz']
+
+patches = ['Subread-2.0.0-GCC-10.2.0-duplicate-symbol.patch']
+
+
+checksums = [  # source
+    'bd7b45f7d8872b0f5db5d23a385059f21d18b49e432bcb6e3e4a879fe51b41a8',
+    # Subread-2.0.0-GCC-10.2.0-duplicate-symbol.patch
+    'cd0024b8c286a327da715749219c7899ce345557ae6d0375407e81bbfa235054']
+
+start_dir = 'src'
+
+buildopts = " -f Makefile.Linux"
```

Comment:
There's a hardcoded `-mtune=core2` in the Makefile, which makes the build fail hard on non-x86 systems.

That's easy to avoid by adding the following (style-wise that should go above this `buildopts = ...` line

```
prebuildopts = "sed -i 's/-mtune=core2//g' Makefile.Linux && "
prebuildopts += "sed -i 's/-mtune=core2//g' longread-one/Makefile && "
```

Reply:
If you reject this in favor of update 2.0.2, I'll check whether this still needs doing.